### PR TITLE
Issue #727: shallow validation of release number

### PR DESCRIPTION
**Pull request by romani**
Issue #727

it is not comprehended validation, but enough to avoid basic typos

### CODE REVIEW

**Comment by romani** [comment]
@stoyanK7 , what IDE you use ?
I have hard problems to build this project in IDEA.

File: releasenotes-builder/src/main/java/com/github/checkstyle/MainProcess.java [hunk 1]
```diff
@@ -103,6 +108,12 @@ public static List<String> validateNotes(Multimap<String, ReleaseNotesMessage> r
         final String errorEnding = "Please correct release number by running https://github.com/"
             + "checkstyle/checkstyle/actions/workflows/bump-version-and-update-milestone.yml";
 
+        if (releaseVersion.split("\\.").length != RELEASE_DIGIT_COUNT) {
+            errors.add("Release number should have 3 digits separated by '.' symbols");
+        }
+        if (StringUtils.countMatches(releaseVersion, ".") != RELEASE_DIGIT_COUNT - 1) {
```
Comment (rnveach) [inline releasenotes-builder/src/main/java/com/github/checkstyle/MainProcess.java:114]: These 2 seem to be the same thing, counting periods. Validation should be regular expression matching `\d+\.\d+\.\d+`.

**Comment by stoyanK7** [comment]
@romani This is what I am using. No issues building/running tests on this PR branch.

```
IntelliJ IDEA 2022.3 (Community Edition)
Build #IC-223.7571.182, built on November 29, 2022
Runtime version: 17.0.5+1-b653.14 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 5.15.0-56-generic
GC: G1 Young Generation, G1 Old Generation
Memory: 1200M
Cores: 8
Non-Bundled Plugins:
    CheckStyle-IDEA (5.69.1)

Kotlin: 223-1.7.20-release-201-IJ7571.182
Current Desktop: ubuntu:GNOME
```

File: releasenotes-builder/src/main/java/com/github/checkstyle/MainProcess.java (hunk 1 above)
Comment (romani) [reply releasenotes-builder/src/main/java/com/github/checkstyle/MainProcess.java]: done.

**Comment by romani** [comment]
I installed latest IDEA, and it works fine now in IDEA.

**Review by rnveach — APPROVED** [review]

**Pull request merged**